nexus/cognitron/web/nuxt.config.js

```diff
@@ -39,12 +39,12 @@ export default {
   ],
 
   publicRuntimeConfig: {
-    meta_api: {
-      url: process.env.NEXUS_COGNITRON_WEB_meta_api.url || 'http://nexus-meta-api:8080'
+    metaApi: {
```

Comment:
In case you wonder, it's because in the file `meta-api.js` the param is defined as `$config.metaApi`